app/code/Magento/Pwa/Test/Mftf/ActionGroup/VeniaCategoryActionGroup.xml

```diff
@@ -15,7 +15,9 @@
             <argument name="categoryName" type="string"/>
             <argument name="productName" type="string"/>
         </arguments>
+        <wait time="3" stepKey="temporaryStaticWait" />
         <click userInput="{{categoryName}}" stepKey="clickOnCategoryName1"/>
+        <wait time="5" stepKey="temporaryStaticWait1" />
```

Comment:
Documented IRL convo: need to experiment with this wait, sometimes fails.

Reply:
Added comment, and moved this line to right place. I ran on my docker env looks good now.